tests/gen_cert_test.py

```diff
@@ -181,3 +216,70 @@ def test_cert_date(self, force_date, locale, timezone, expected):
         """
         cert_date = get_cert_date(force_date, locale, timezone)
         assert_equal(expected, cert_date)
+
+
+class CertificateSubtemplateTests(unittest.TestCase):
+    """
+    Tests the subtemplates feature for certs
+
+    Subtemplates are currently written to key on designations only
+
+    These tests have a specific order but do not need to be run in order.
```

Comment:
The comment was put there to flag the tests as being odd.  Which worked beautifully with the exception that you saw the comment and refused to review the tests.  Ah the best laid plans...We'll remove the line when the tests are fixed.  Or it stays so people know there is something fishy going on here.

Running them in a different order works but does not make sense because if the 3rd test, test_subtemplates, passes then the other two tests must be true.  

I know that tests should not be ordered.  But also that tests should not be too complex.  The current test are clear.  Flawed, yes, but not broken.  Again, is it worth the effort?